internal/conf/config.yaml

```
@@ -86,12 +86,15 @@ realtime:
     id: ""                # birdweather ID
 
-  openweather:
-    enabled: false
-    apikey: ""        # OpenWeather API key
-    endpoint: "https://api.openweathermap.org/data/2.5/weather" # OpenWeather API endpoint
-    units: metric     # metric or imperial
-    language: en      # language code
+  weather:
+    provider: yrno
+    pollinterval: 60
+    debug: false  
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)